components/authz-service/storage/v2/memstore/memstore.go

```diff
@@ -339,6 +339,11 @@ func (s *State) DeleteRule(ctx context.Context, id string) error {
 	return nil
 }
 
+func (*State) ApplyStagedRules(context.Context) error {
+	// TODO
```

Comment:
should this TODO be done before this gets merged?

Reply:
I think it can be removed.